test/room_test.rb

```diff
@@ -1,24 +1,26 @@
+require 'minitest'
 require 'minitest/autorun'
 require 'minitest/pride'
 require './lib/room'
 
+
 class RoomTest < Minitest::Test
   def test_it_exists
-    room = Room.new(:bedroom, 10, '13')
+    room = Room.new(:bedroom, 10, 13)
     assert_instance_of Room, room
   end
 
   def test_it_has_a_category
-    skip
-    room = Room.new(:bedroom, 10, '13')
+    room = Room.new(:bedroom, 10, 13)
     assert_equal :bedroom, room.category
   end
 
   def test_it_can_get_area
-    skip
-    room1 = Room.new(:bedroom, 10, '13')
-    room2 = Room.new(:living_room, 15, '12')
+    room1 = Room.new(:bedroom, 10, 13)
+    room2 = Room.new(:living_room, 15, 12)
     assert_equal 130, room1.area
     assert_equal 180, room2.area
   end
```

Comment:
We would have liked to see you add tests in this file to test the first iteration of the interaction pattern.